src/components/Votes.jsx

```diff
@@ -0,0 +1,32 @@
+import { updateVotes } from "../utils/api"
+import {useState} from "react"
+import "../css/Votes.css"
+
+export default function Votes({articleId, setIndividualArticle}) {
+const [error, setError] = useState(null)
+
+function handleClick(event) {
+    let increment
+    if(event.target.innerText === "Vote 👍") increment = 1
+    else{increment = -1}
+
+    const votes = {inc_votes: increment}
+    updateVotes(articleId, votes)
+    .then(() => {setError(null)})
+    .catch((error) => {
+        setIndividualArticle((currArticle) => {
+            return {...currArticle, votes: currArticle.votes - increment}
+        })  
+        setError("likes not available")
+    })
+    setIndividualArticle((currArticle) => {
+        return {...currArticle, votes: currArticle.votes + increment}
+    })
+}
```

Comment:
You may want to make it so each user can only upvote or downvote, and only once (much like a upvote or downvote on Reddit or Youtube). But the user experience is up to you!